com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/resolvers/PropertiesResolver.java

```diff
@@ -197,7 +192,7 @@ private Object resolveKeyInIterable(String key, Collection<?> listOfProperties,
         if (StringUtils.isNumeric(key)) {
             try {
                 return IterableUtils.get(listOfProperties, Integer.parseInt(key));
-            } catch (IndexOutOfBoundsException e) {
+            } catch (IndexOutOfBoundsException ignored) {

```

Comment:
Please retain the E as a standard exception name and add a comment in the catch block explaining why do you ignore the exception

Reply:
I don't know. It was just ignored before :D

Reply:
Then return the previous name, and write //Intentionally ignored